### PR TITLE
Add JSON schema endpoint

### PR DESCRIPTION
**Pull request by edgarmueller**
Add JSON schema endpoint which returns the schema of a given instance via `api/v1/schema/:modelUri`

### CODE REVIEW

File: com.eclipsesource.modelserver.common/pom.xml [hunk 1]
```diff
@@ -1,7 +1,19 @@
 <project xmlns="http://maven.apache.org/POM/4.0.0"
-	xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
-	xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+		 xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+		 xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
 	<modelVersion>4.0.0</modelVersion>
+	<build>
```
Comment (tortmayr) [inline com.eclipsesource.modelserver.common/pom.xml]: Java Version can be set with the <java.version> property. It's not necessary to explicitly configure the plugin.

In this case, it's probably better to set the version in the parent pom.

**Comment by tortmayr** [comment]
Travis build fails because of the switch to JDK9.  The .travis.yml needs to be updated accordingly
(I'd recommend to build with openjdk11)

**Comment by edgarmueller** [comment]
@tortmayr Thanks for the review, I've updated the parts you've outlined

**Review by tortmayr — APPROVED** [review]
The code looks good to me. The schema generation seems to work fine but seems to be incomplete. I guess this is because of the missing mappings for some ecore features?

The upgrade to jdk9/11 might cause the maven snapshot deployment to fail. I had a similar issue with GLSP the other day. If this is the case I will create a follow-up PR to fix CI.

**Comment by edgarmueller** [comment]
Yes, that's true, it's incomplete right now, but it should serve as a good starting point. Thanks for taking care of creating a follow-up issue, should the deployment fail.

**Pull request merged**